xbmc/games/addons/GameClientInGameSaves.h

```diff
@@ -0,0 +1,73 @@
+/*
+ *      Copyright (C) 2016-2017 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this Program; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#include "addons/kodi-addon-dev-kit/include/kodi/kodi_game_types.h"
+
+#include <string>
+
+struct GameClient;
+
+namespace GAME
+{
+  class CGameClient;
+
+  /*!
+   * \brief This class implements in-game saves.
+   *
+   * Some games do not implement state persistence on their own, but rely on the frontend for saving their current
```

Comment:
brief ends when starting a new paragraph (see https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmddetails and http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#cppblock). but why not be more specific